pom.xml

```diff
@@ -15,34 +15,34 @@
 
 	<properties>
 		<java.version>21</java.version>
-
 		<main.class>uk.gov.companieshouse.charges.data.ChargesDataApiApplication</main.class>
 
+		<!-- internal dependencies -->
+		<spring.boot.version>3.3.5</spring.boot.version>
```

Comment:
Can this be 3.4.0?

Reply:
And the parent pom version can be 2.1.10 I think

Reply:
Resolved in latest commit